tests/neg-deep-subtype/1828.scala

```diff
@@ -2,10 +2,12 @@
 
 class Test {
   def remove[S](a: S | Int, f: Int => S):S = a match {
-    case a: S => a             // error
+    case a: S => a             // warn
```

Comment:
This tag is not checked by the testing framework. This implies that this file in not testing anything. This implies that all tests that do not have a test file are broken.

We need to make sure that we check the position and number of warnings emitted under `-Xfatal-warnings`/`-Werror`. Our assumption when designing those tests was that we would check them as errors.

Reply:
Dale recently added that feature, it should be working (at least it's in Vulpix in ParallelTesting). I will make sure it works.

Reply:
Oh. Now I see. It won't test both